draft-ietf-mops-streaming-opcons.md

```diff
@@ -483,35 +485,39 @@ Here are a few common resolutions used for video content, with typical ranges of
 
 ## Path Bandwidth Constraints {#sec-band-constraints}
 
 Even when the bandwidth requirements for video streams along a path are well understood, additional analysis is required to understand the constraints on bandwidth at various points in the network. This analysis is necessary because media servers may react to bandwidth constraints using two independent feedback loops:
 
-* Media servers often respond to application-level feedback from the media player that indicates a bottleneck somewhere along the path by adjusting the amount of media that the media server will send to the media player in a given timeframe. This is described in greater detail in {{sec-abr}}.
+* Media servers often respond to application-level feedback from the media player that indicates a bottleneck somewhere along the path by adjusting the number of media segments that the media server will send to the media player in a given timeframe. This is described in greater detail in {{sec-abr}}.
 
-* Media servers also typically rely on transport protocols with capacity-seeking congestion controllers that probe for bandwidth and adjust the sending rate based on transport mechanisms. This is described in greater detail in {{sec-trans}}.
+* Media servers also typically rely on transport protocols with capacity-seeking congestion controllers that probe for available path bandwidth and adjust the media segment sending rate based on transport mechanisms. This is described in greater detail in {{sec-trans}}.
 
 The result is that these two (potentially competing) "helpful" mechanisms each respond to the same bottleneck with no coordination between themselves, so that each is unaware of actions taken by the other, and this can result in QoE for users that is significantly lower than what could have been achieved.
 
-One might wonder why media servers and transport protocols are each blissfully ignorant of what the other is doing, and there are multiple reasons for that. However, one reason is that media servers may be implemented to run under a general-purpose operating system that typically has its transport protocols implemented in the operating system kernel, making decisions that the media server never knows about.
+One might wonder why media servers and transport protocols are each so blissfully ignorant of what the other is doing, and there are multiple reasons for that. One reason is that media servers are often implemented as applications executing in user space, relying on a general-purpose operating system that typically has its transport protocols implemented in the operating system kernel, making decisions that the media server never knows about.
```

Comment:
We probably should cut 'blissfully' here too.  It's a bit sarcastic, now that I read it again...